### DISCUSSION TITLE
New AMD Vulnerability: "Take A Way"

### DISCUSSION THREAD

**Discussion by owlshrimp** [speed47/spectre-meltdown-checker#546] [Other vulnerabilities / information]
Hi,

There's a new AMD processor vulnerability targeting the L1 cache way predictor.
https://www.amd.com/en/corporate/product-security
https://www.zdnet.com/article/amd-processors-from-2011-to-2019-vulnerable-to-two-new-attacks/

Not much word yet on mitigations from AMD. The authors speculate in their paper that software mitigation may be possible on a per-application basis, but proper mitigation may require chicken bits or new microcode. (if it's possible for microcode to mitigate the issue or such chicken bits exist)

**Comment by speed47** [comment]
We'll see if AMD revise their stance on this, as for now the say:

> AMD believes these are not new speculation-based attacks.

Keeping this open in case it changes at some point.

**Comment by speed47** [comment]
In the end, AMD never changed their stance about this one: there's no CVE, no microcode or mitigation to look for. Closing this to cleanup the ticket queue.
